infra/src/99_variables.tf

```diff
@@ -31,4 +31,72 @@ variable "tags" {
   }
 }
 
+variable "vnet_address_space" {
+  type    = list(string)
+  default = ["10.10.0.0/20"]
+}
+
+variable "snet_pendpoints_address_spaces" {
+  type    = list(string)
+  default = ["10.10.3.0/24"]
+}
+
+
+variable "cidr_subnet_fnsubscriptionasync" {
+  type    = list(string)
+  description = "The subscription async function address space"
+}
+
+variable "cidr_subnet_fnsubscription" {
+  type    = list(string)
+  description = "The subscription function address space"
+}
```

Comment:
please @Krusty93 can you check addresses in order to avoid overlap with other VNET address spaces?